app/styles/site-views/form-view.scss

```diff
@@ -9,3 +9,17 @@
   font-size: $text-display-size;
    margin-top: $footer-display-margin-top;
 }
+
```

Comment:
.form-display-truncate should have nested children. please make sure you follow the scss guidelines for nesting.

Reply:
corrected